hangman.py

```diff
@@ -0,0 +1,80 @@
+#Python program to illustrate Hangman 
+
+#Python modules
+import random
+from collections import Counter
+
+#List of Fruits to guess from
+some_words = ['apple', 'banana', 'mango', 'strawberry', 'orange', 'grape', 'pineapple', 'apricot', 'lemon', 'coconut', 'watermelon', 'cherry', 'papaya', 'berry', 'peach', 'lychee', 'muskmelon']
+
+#randomly choose a secret word from our "some_words" LIST
+word = random.choice(some_words)
+
+if __name__ == '__main__':
+    print("Guess the word! HINT: word is a name of a fruit")
+
+    for i in word:
+        print("_", end=' ')  # Pritnting the empty spaces for letters of the word
+
+    playing = True
+
+    letter_guessed = ''
+    chances = len(word) + 2
+    correct = 0
+    flag = 0
+
+    try:
+        while (chances != 0) and (flag == 0):
+            print()
+            chances -= 1
+
+            try:
+                guess = str(input("Enter a letter to guess: "))
```

Comment:
Currently, the game is case-sensitive, so 'A' and 'a' are treated differently. 
Normalize both the secret word and user input
 (e.g., using .lower()) to make the game case-insensitive.